### ISSUE TITLE
Is it in scope of CNTT, Openstack dynamically provides node resources for Kubernetes cluster?

### ISSUE THREAD

**Issue by YuPengZTE** [cntt-n/CNTT#1334]
The node is dynamically created by OpenStack when it is found that the node resources are insufficient.
@CsatariGergely  @ulikleber 

P.S.
If it is in scope, the flow would be changed when creating pod by Kubernetes.

As shown in the following figure, when the pod is created, the node is found to be insufficient, in step "3. Req Node", the virtual machine node is created through "infrastructure management" (the virtual machine provided by OpenStack). 
This flow is different from the native Kubernetes.

![PV Network Node Manager](https://user-images.githubusercontent.com/20062886/77039003-08dd2680-69f0-11ea-94dd-5ed3db130aea.png)

**Comment by CsatariGergely** [comment]
The interaction between the Cloud Providers and the actual clouds is not described in RA2. Also I think that the supported Cloud Providers are not in the scupe of RA2 as they are not visible for the VNF-s.

**Comment by YuPengZTE** [comment]
@CsatariGergely, thanks.
yes,  they are not visible for the VNF-s,  they are integrated.

 ```Most prominently Kubernetes Cloud Provider collect various integrations towards a particular infrastructure platform e.g. to create volumes or additional worker nodes. The Kubernetes cluster api harmonizes some of the k8s cluster management related integrations with different infrastructure platforms under a common API``` in ![RA2 relation to underlying infrastructure](https://github.com/cntt-n/CNTT/raw/master/doc/ref_arch/figures/ref_arch_ra2_relationships.png)

Is the integration method not in the scope of CNTT?

**Comment by xmulligan** [comment]
So you are basically asking if ClusterAPI or something similar should be part of CNTT?

**Comment by YuPengZTE** [comment]
Yes, similar to Kubernetes cloud provider, but this cloud is a private cloud, which is provided by OpenStack.
